### PR TITLE
Hotfix: Deactivate servicelogger entirely

### PR DESCRIPTION
**Pull request by timotree3**

### CODE REVIEW

File: profiles/logical/hpos/default.nix [hunk 1]
```diff
@@ -273,10 +273,6 @@ in
          app_id = "core-app";
          bundle_url = "https://holo-host.github.io/holo-hosting-app-rsm/releases/downloads/v0.1.1-alpha2/core-app-skip-proof.happ";
        }
-       {
-         app_id = "servicelogger";
-         bundle_url = "https://holo-host.github.io/servicelogger-rsm/releases/downloads/v0.1.0-alpha11/servicelogger.happ";
-       }
```
Comment (robbiecarlton) [inline profiles/logical/hpos/default.nix:279]: What's the value of removing this in addition to disabling above?

**Review by robbiecarlton — APPROVED** [review]

File: profiles/logical/hpos/default.nix (hunk 1 above)
Comment (timotree3) [reply profiles/logical/hpos/default.nix:279]: The disable feature only disables the per hosted happ servicelogger, but does still passes through core-happs transparently

**Comment by alastairong** [comment]
We merged #978 instead

**Comment by alastairong** [comment]
@timotree3 Should we close this?

**Pull request closed without merging**